### DISCUSSION TITLE
Merge two tenants DB

### DISCUSSION THREAD

**Discussion by felixfenu** [snipe/snipe-it#12620] [General]
Hi Everyone!
I am in the process of merging two Snipe-IT databases (two different tenants into one) and I would like to know what are the best practices to do it.
What is the recommended process? Especially thinking about FK in database
Eg. Manufacturer id 1 in tenant 1 is Dell, but
Manufacturer id 1 in tenant 2 is Lenovo

And so also with the other tables

**Comment by anhtran917** [comment]
To merge two Snipe-IT databases, you should first back up both databases and then import the data from one database into the other. You will need to update any foreign key references to ensure that they point to the correct records in the merged database. 
It is recommended to consult the Snipe-IT documentation and seek advice from the Snipe-IT community for best practices and guidance on this process.

**Comment by snipe** [comment]
There is not an easy way to do this without losing some amount of history :( The foreign keys would be a nightmare to try to correct. If you don't care about history, export via CSV the one you want to merge INTO the other one. Other than that, it would be a pretty miserable process prone to failure :(

**Reply by anhtran917** [reply]
Okay, thanks.
